content/shmem_malloc.tex

```diff
@@ -62,21 +62,21 @@
 
 \apireturnvalues{
     The \FUNC{shmem\_malloc} routine returns a pointer to the allocated space;
-    otherwise, it returns a \CONST{NULL} pointer.
+    otherwise, it returns a null pointer.
     
     The \FUNC{shmem\_free} routine returns no value.
     
     The \FUNC{shmem\_realloc} routine returns a pointer to the allocated space
-    (which may have moved); otherwise, it returns a null pointer.
+    (which may have moved); otherwise, it returns a \const{NULL} pointer.
```

Comment:
The original text was correct. It should read: "otherwise, it returns a null pointer"

Reply:
Still there